src/AutoRotatingCarousel.js

```diff
@@ -157,7 +158,7 @@ class AutoRotatingCarousel extends Component {
       <Carousel
         autoplay={open && autoplay}
         className={classes.carousel}
-        containerStyle={{ height: '100%' }}
+        containerStyle={{ height: containerHeight || '100%' }}
```

Comment:
This prop is far too specific. We should add a `containerStyle` prop instead and spread it into the Carousel's `containerStyle`, like this: `containerStyle={{ height: '100%', ...containerStyle }}`

Also, please add the new prop to the `propTypes` below.